packages/hub/schema/group.js

```diff
@@ -51,5 +36,39 @@ module.exports = class Group {
       }
     });
   }
+};
+
+const validatedFieldFilters = function(allFields,  searchQuery, model) {
```

Comment:
I made these pure functions outside of the class, but let me know if they should be functions that are part of the class

Reply:
I personally also prefer pure functions outside of classes, and aggressively move functions out of classes when I can.